Formula/gitmoji.rb

```diff
@@ -1,8 +1,8 @@
 require "language/node"
 
 class Gitmoji < Formula
-  desc "Emoji guide for your commit messages"
-  homepage "https://gitmoji.carloscuesta.me"
+  desc "Interactive command-line tool for using gitmoji on commits"
```

Comment:
Is this intentionally `gitmoji` and not `emoji`? The repository description is also the old description so I'm unsure why we want to change it (seems clear as-is).

Reply:
The description that is right now, refers to the gitmoji repository.

This formula is not an emoji guide, is a command line tool for using gitmoji on git commits that’s why I considered that the description should be updated 🙂

Btw, I’m not sure who added this first description (not me) but in my opinion is not accurate to what the gitmoji-cli tool is doing

Reply:
I wanted to use the same description that I’m using in: https://github.com/carloscuesta/gitmoji-cli

But it was not possible as descriptions can’t start with an article or the formula name so I had to tweak it a little bit

Reply:
I think we can use:
```suggestion
  desc "Interactive command-line tool for using emoji in commit messages"
```
(Apologies for the delayed response.)